### ISSUE TITLE
Host cyphologi on teach.mozilla.org

### ISSUE THREAD

**Issue by brettgaylor** [MozillaFoundation/encryption_campaign#216]
To help ensure that this app is framed as an educational tool vs. a secure communications tool, we'd like to host this on teach.mozilla.org.

@simonwex what would this entail?

**Comment by brettgaylor** [comment]
And could we add this to our next heartbeat?

**Comment by davidascher** [comment]
My suggestion:

1) advocacy team is on the hook for understanding how to create a PR for
the teach website, including understanding the IA of the site (by talking
to teach team, design, etc.).
2) teach team owns the site, so is on the hook for review & merge / push to
prod

I'm all for collaboration and I see the point in comment #0, but want to
clarify that teach team is accountable for teach site -- hosting the page
has to make sense for them too, in the context of a community, a website
with an IA, etc.  In particular when it comes to scheduling impact, dealing
with making it "make sense" may cause a traffic jam.

**Comment by brettgaylor** [comment]
Makes sense - and if we aren't able to find cycles or it doesn't make sense for other reasons it won't block us from shipping the tool.
